.github/actions/build-rpm/src/main.ts

```diff
@@ -0,0 +1,140 @@
+import * as core from '@actions/core'
```

Comment:
This is the main file that is the entrypoint to the local github action for building rpm. This contains steps to setup rpm environment, build and sign the rpm package